src/test/integration/EthTokensProviderTest.kt

```diff
@@ -0,0 +1,37 @@
+package integration
+
+import main.CONFIG
+import main.ConfigKeys
+import notary.db.tables.Tokens
+import org.jooq.impl.DSL
+import java.sql.DriverManager
+import org.junit.jupiter.api.Test
+
+/**
+ * Test Postgres ethereum ERC20 tokens provider
+ */
+class EthTokensProviderTest {
+
+    /**
+     * List all tokens in database
+     */
+    //    @Test
```

Comment:
Looks like it is not a test

Reply:
It's a way to disable test in CI, but run it manually.